otter/convergence/service.py

```diff
@@ -344,7 +341,11 @@ def converge_all_groups(log, currently_converging, my_buckets, all_buckets,
     # convergence manually when it's spiraling out of control.
     # https://github.com/rackerlabs/otter/issues/1215
 
-    group_infos = yield get_my_divergent_groups(my_buckets, all_buckets)
+    if divergent_flags is None:
```

Comment:
Yeah, I could perhaps continue refactoring this up the chain... I thought it was *kinda* nice to leave all the ZK interaction in `converge_all_groups`, since it now does both the `GetChildren` and the `Stat`, but I could refactor it up further so that `buckets_acquired` gets it.